src/Tasks/Microsoft.NETFramework.props

```diff
@@ -43,4 +43,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
    <Import Project="$(NetFrameworkPropsPath)" />
 
+   <!--Import props that are common for both full framework and core-->
+   <Import Project="$(MSBuildToolsPath)\Microsoft.NET.props" />
```

Comment:
Should this be here or in `CurrentVersion`? I don't know if anyone's importing that directly; if they were this would stop setting the moved things.

Reply:
This is the only place it is imported, but I think you are right, it is easier to understand putting it in CurrentVersion. And it will help the future case